sys/auto_init/auto_init.c

```diff
@@ -90,6 +94,9 @@ void auto_init(void)
     config_load();
 #endif
 
+#ifdef MODULE_TINYMT32
+    genrand_init(0);
```

Comment:
I honestly don't know. Without any seeding, tinymt32 always returns "0". I assume "0" is as good as seed as any number.